.gitignore

```diff
@@ -30,6 +30,9 @@ _tools/
 debug
 debug.test
 
+# Track src/x/debug though
+!src/x/debug
```

Comment:
Hah, that's frustrating.  

I think that `debug.test` is usually what we see end up in our directories, perhaps we should just remove the `debug` entry (and keep `debug.test`) instead of explicitly opt in this path?

Just mindful and hoping we wouldn't have to do this in the future for other packages with similar names.

Reply:
I removed `debug` as well as the explicit opt `src/x/debug` and it seems alright alright